build_centreon_plugins_32.bat

```diff
@@ -1,6 +1,6 @@
 @ECHO OFF
 
-SET VERSION_PLUGIN=20000101
+SET VERSION_PLUGIN=20211102

```

Comment:
Don't change `VERSION_PLUGIN` it is automatically updated with proper version at build time.